extension/data/deinflect.dat

```diff
@@ -363,3 +360,6 @@ past negative
 でございましょう	でございます	8320	18
 でございまして	でございます	8320	21
 であれば	でございます	8320	17
+いって	いく	384	0
```

Comment:
Ah, sorry for the misunderstanding but the index isn't into the list of similar inflections but into the reason list at the top of the file. That means, we should keep the final index numbers the same since the conjugation reasons are the same.

(14, 21, 3)